### PR TITLE
Slack message integration

### PR DESCRIPTION
**Pull request by poi33**
Added a feature to the office-league that can post messages to slack with webhooks. 
League needs to have the property `slackIntegration` boolean and true
Should post messages on game start and game finished

### CODE REVIEW

File: src/main/resources/lib/office-league-slack.js [hunk 1]
```diff
@@ -15,15 +15,18 @@ function createFinishedGameMessage(data) {
     };
 
     let matchResult = "";
```
Comment (reisfmb) [inline src/main/resources/lib/office-league-slack.js:17]: Why did you initialize it like that? it's value will always be replaced in one of the two cases in the conditional at line 20, so isn't it better to initialize it like `let matchResult;`?

File: src/main/resources/lib/office-league-slack.js [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+            },
+            createPlayerSection(sides.blue[0])
+        );
+    }
+
+    return message;
+}
+
+/**
+ * Method for formating the expted message
```
Comment (reisfmb) [inline src/main/resources/lib/office-league-slack.js:123]: expted -> expected

**Review by reisfmb — APPROVED** [review]
For me the code is :+1: 

Just added two quick suggestions.

File: src/main/resources/lib/office-league-slack.js (hunk 1 above)
Comment (poi33) [reply src/main/resources/lib/office-league-slack.js:17]: I think i did matchResult += in a previus version.

**Pull request merged**